### PR TITLE
flavours: update to 0.7.1.

### PR DESCRIPTION
**Pull request by Newchair2644**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64)

### CODE REVIEW

File: srcpkgs/flavours/template [hunk 1]
```diff
@@ -9,7 +9,7 @@ maintainer="Numair Dubas <Numair2644@protonmail.com>"
 license="MIT"
 homepage="https://github.com/Misterio77/flavours"
 distfiles=${homepage}/archive/refs/tags/v${version}.tar.gz
```
Comment (classabbyamp) [inline srcpkgs/flavours/template:11]:
```suggestion
distfiles=https://github.com/Misterio77/flavours/archive/refs/tags/v${version}.tar.gz
```

File: srcpkgs/flavours/template [hunk 2]
```diff
@@ -9,7 +9,7 @@ maintainer="Numair Dubas <Numair2644@protonmail.com>"
 license="MIT"
 homepage="https://github.com/Misterio77/flavours"
```
Comment (classabbyamp) [inline srcpkgs/flavours/template:10]:
```suggestion
homepage="https://github.com/Misterio77/flavours"
changelog="https://raw.githubusercontent.com/Misterio77/flavours/master/CHANGELOG.md"
```

File: srcpkgs/flavours/template (hunk 1 above)
Comment (Newchair2644) [reply srcpkgs/flavours/template:12]: Thank you for the suggestions. Should I squash the commits?

Comment (classabbyamp) [reply srcpkgs/flavours/template:12]: yes please

Comment (Newchair2644) [reply srcpkgs/flavours/template]: Done. I also used an email I found on your packages because `xlint` was complaining about a noreply address.

**Pull request closed without merging**